doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2044,6 +2044,28 @@ The issues in this section are related to the :ref:`nrf_desktop` application.
     nRF Desktop is also affected by :ref:`hogp_readme` library's issue :ref:`NCSDK-30288 <ncsdk_30288>`.
     The issue may cause accessing data under ``NULL`` pointer in case of Bluetooth disconnection while forwarding a configuration channel operation in the :ref:`nrf_desktop_hid_forward`.
 
+.. rst-class:: v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0 v2-7-0
+
+NCSDK-30504: USB High-Speed slight report rate drop
+  Due to interoperability issues with Bluetooth, the nRF Desktop USBHS report rate is slightly lower than 8000 Hz.
+  The device is sometimes too slow to provide the report in time, which causes a report rate drop up to around 7997 Hz.
+
+  **Affected platforms:** nRF54H20
```

Comment:
```
NCSDK-30504: USB High-Speed slight HID report rate drop
  While Bluetooth is in use, the device is sometimes too slow to provide the HID report over USB in time, which causes HID report rate drop to around 7997 Hz.
  The slight HID report rate drop over USB High-Speed might be visible for example during Bluetooth LE advertising.

  **Affected platforms:** nRF54H20
```

Reply:
I would not call this `interoperability issue`. It might be that we are simply too slow to handle both.